packages/slider/src/Slider.ts

```diff
@@ -78,18 +103,29 @@ export class Slider extends Focusable {
         return this._variant;
     }
 
+    public get handleName(): string {
+        return 'value';
+    }
+
     /* Ensure that a '' value for `variant` removes the attribute instead of a blank value */
     private _variant = '';
 
     @property({ attribute: false })
-    public getAriaValueText: (value: number) => string = (value) => `${value}`;
+    public getAriaValueText: (values: Map<string, string>) => string = (
+        values
+    ) => {
+        const valueArray = Array.from(values.values());
+        if (valueArray.length === 2)
+            return `${valueArray[0]} - ${valueArray[1]}`;
+        return valueArray.join(', ');
+    };
 
     @property({ attribute: false })
```

Comment:
Not your code, but not sure why this is `@property`...

Reply:
Will try removing the `@property`